package.json

```diff
@@ -14,7 +14,7 @@
     "ganache:start": "npx ganache-cli -m 'beach swap combine paper music cook electric bullet trust actress liquid asthma' -l 1000000000 > /dev/null 2>&1 &",
     "ganache:stop": "kill `ps ax|grep ganache-cli|grep -v grep| awk '{print $1}'` 2> /dev/null; exit 0",
     "truffle:migrate": "npx lerna run migrate --scope tasit-contracts",
-    "pretest": "npm run bootstrap && npm run ganache:stop && npm run ganache:start && npm run truffle:migrate",
```

Comment:
I've used that only for dev env, to ensure that all installs and links are OK only running `npm test`, but since Decentraland repo the bootstrapping is slower.
We will gain some seconds on CI too because on our config we have the bootstrap step before test step (bootstrap was running twice).

Reply:
Ah ok, great. Resolving this.